packages/databricks-vscode/resources/python/dbconnect-bootstrap.py

```diff
@@ -17,17 +20,29 @@ def load_env_file_from_cwd(path: str):
             os.environ[key] = value
     return
 
-load_env_file_from_cwd(os.getcwd())
+
+script = sys.argv[1]
+sys.argv = sys.argv[1:]
+logging.debug(f"Running ${script}")
+logging.debug(f"args: ${sys.argv[1:]}")
+
+try: 
+    cur_dir = os.path.dirname(script)
+except Exception as e:
+    logging.error(f"Failed to get current directory: {e}")
+    cur_dir = os.getcwd()
+
+root_dir = os.getcwd()
+load_env_file_from_cwd(root_dir)
 
 log_level = os.environ.get("DATABRICKS_VSCODE_LOG_LEVEL")
 log_level = log_level if log_level is not None else "WARN"
 
-import logging
 logging.basicConfig(level=log_level)
 
 db_globals = {}
 
-from databricks.sdk.runtime import dbutils
+from databricks.sdk.runtime import dbutils  # noqa: E402
```

Comment:
Why can't we move the import at the top? The environment must be setup before the import?

Reply:
yep